src/architecture/android.md

```diff
@@ -0,0 +1,33 @@
+# Android app architecture
+
+The implementation of the Android app is divided into a number of components:
+* the shared Android/OpenHarmony implementation of `servoshell` (`ports/servoshell/egl`)
+* the Android-specific `servoshell` integration (`ports/servoshell/egl/android.rs`)
+* the main Android activity (`support/android/apk/servoapp/src/main/java/org/servoshell`)
+* the ServoView component (`support/android/apk/servoview/src/main/java/org/servo/servoview`)
+  * the Android SurfaceView (`ServoView.java`)
+  * the Servo library wrapper (`Servo.java`)
```

Comment:
Maybe this can be
```suggestion
  * the Servo engine wrapper (`Servo.java`)
```

as later we use that term in 'The lowest level is the Servo engine wrapper, which implements ...`.